getting-started/setup-multi-node-auth.md

```diff
@@ -1,20 +1,113 @@
-# Authentication setup for multi-node
+# Setting up a self-managed multi-node environment
 
-Beyond the [basic setup][] for multi-node TimescaleDB, there are a number of 
-important aspects of the implementation that may need to be configured,
-especially for a production environment:
-- Enabling secure communication between the access node and the data 
-nodes
-- Adding additional users/roles and permissions to interact with the system
+If you have multiple instances of PostgreSQL running, you can set them up to
+act as a TimescaleDB multi-node environment.
+
+To start, you'll need to have the following:
+- One PostgreSQL instance to act as an access node
+- One or more PostgreSQL instances to act as data nodes
+- TimescaleDB [installed][install] and [set up][setup] on all nodes
+- Access to a superuser role (e.g. `postgres`) on all nodes
+
+## Multi-node configuration
```

Comment:
Does this belong on a page around "_authentication_"?  Just asking if this is right place here?  (Also, what does timescaledb-tune configure it too?)

if tune configures, do we need this?  And instead add a mention of this to the main configuration page (which has a section about important GUCs _for postgres) and/or troubleshooting?

Reply:
I struggled a bit with this section.  The `max_prepared_transactions` is required for multi-node, and is not set by timescaledb-tune.  So really this section only exists to tell users to set this.

I think mentioning tune here is a bit confusing, I'll just change this to [normal configuration for TimescaleDB][link to configuration page].